.github/workflows/supply-chain-security-validation.yaml

```diff
@@ -76,12 +76,12 @@ jobs:
       packages: read
     steps:
       - name: Check out repository
-        uses: actions/checkout@v4
-      - uses: fabasoad/setup-enry-action@main
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # pin@v4
+      - uses: fabasoad/setup-enry-action@84a938519217e730824fd7b4b1fb12b52471b22d # pin@main
```

Comment:
I agree that we should not be depending on the head of the main branch, but pinning to a commit will trigger a lot of updates.

Reply:
this is true, but the on the other end there is a security issue. We are giving actions access to many resources in gh. Git tags are not immutable